lms/static/js/spec/bookmarks/bookmarks_spec.js

```diff
@@ -0,0 +1,134 @@
+define(['backbone', 'jquery', 'underscore', 'js/common_helpers/ajax_helpers', 'js/common_helpers/template_helpers',
+        'js/bookmarks/models/bookmark',
+        'js/bookmarks/collections/bookmarks',
+        'js/bookmarks/views/bookmarks_button',
+        'js/bookmarks/views/bookmarks_list',
+        'moment'
+       ],
+    function (Backbone, $, _, AjaxHelpers, TemplateHelpers, BookmarksModel, BookmarksCollection, BookmarksButtonView,
+              BookmarksListView) {
+        'use strict';
+
+        describe("lms.courseware.bookmarks", function () {
+
+            beforeEach(function () {
+                loadFixtures('js/fixtures/bookmarks/bookmarks.html');
+                TemplateHelpers.installTemplates(
+                    [
+                        'templates/message_view',
+                        'templates/bookmarks/bookmarks_list'
+                    ]
+                );
+            });
+
+            describe("Bookmarks", function () {
+                var bookmarksButtonView;
+
+                beforeEach(function () {
+                    bookmarksButtonView = new BookmarksButtonView({});
+                    var show = true;
+                    var fakeBookmarksShown = function () {
+                        show = !show;
+                        return show;
+                    };
+                    spyOn(bookmarksButtonView.bookmarksListView, 'isVisible').andCallFake(fakeBookmarksShown);
+                });
+
+                var createBookmarksData = function () {
+                    return {
+                        results: [
+                            {
+                                id: 0,
+                                display_name: "A Global History of Architecture: Part 1",
+                                created: "2014-09-23T14:00:00Z",
+                                course_id: "MITx/4.605x_2/3T2014",
+                                usage_id: "i4x://RiceX/BIOC300.1x/openassessment/cf4c1de230af407fa214905b90aace57",
+                                path: [
+                                    {
+                                        display_name: "Week 1",
+                                        usage_id: "i4x://RiceX/BIOC300.1x/chapter/cf4c1de2efmveoirm1490e57"
+                                    },
+                                    {
+                                        display_name: "Reflection",
+                                        usage_id: "i4x://RiceX/BIOC300.1x/sequential/foivmeiormoeriv4905b90aace57"
+                                    }
+                                ]
+                            }
```

Comment:
I think we should add a couple of more items to this list and check that rendering works correctly.